src/doc/rustc/src/platform-support/aarch64-apple-darwin.md

```diff
@@ -0,0 +1,41 @@
+# `aarch64-apple-darwin`
+
+**Tier: 2 with host tools**
+
+64-bit ARM-based Apple devices running macOS, Macs with M1 or M2 Apple Silicon chips. M3 or higher is [`arm64e-apple-darwin`](platform-support/arm64e-apple-darwin.md).
```

Comment:
This isn't accurate; `aarch64-apple-darwin` works just fine on M3 too (I have one) and `arm64e-apple-darwin` works "just fine" (module ongoing ABI stability and implementation issues) on all M-series chips identically.  The distinction between the targets is the use of PAuth in the function call ABI, but PAuth is implemented on all of these CPUs and Apple's `arm64e` ABI is available on all of Apple's Arm-based devices, not just the Macs.

```suggestion
64-bit ARM-based Apple devices running macOS: Macs with M1-family or later Apple Silicon CPUs.
```

Reply:
That was a misunderstanding on my part. Do you happen to know who the maintainers are?

Reply:
Sadly, I don't; I would love to know myself.  :(